src/main/java/io/kestra/plugin/aws/eventbridge/PutEvent.java

```diff
@@ -0,0 +1,138 @@
+package io.kestra.plugin.aws.eventbridge;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.kestra.core.exceptions.IllegalVariableEvaluationException;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.models.tasks.VoidOutput;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.aws.AbstractConnection;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import software.amazon.awssdk.http.apache.ApacheHttpClient;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.eventbridge.EventBridgeClient;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequest;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequestEntry;
+
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Plugin(
+        examples = {
+                @Example(
+                        title = "Put an event from a map.",
+                        code = {
+                                "eventBusName: \"events\"",
+                                "eventSource: \"Kestra\"",
```

Comment:
```suggestion
                                "source: \"Kestra\"",
```

the property should be named `source`, as this is used in EventBridge and it's required event pattern in EventBridge rules 
![image](https://github.com/kestra-io/plugin-aws/assets/86264395/37cd47a0-f9e3-4259-bac9-d3672836b5c0)